src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
```

Comment:
Consider using named constants for the magic numbers used in the `rand.nextInt` methods to improve readability and maintainability.